AUS/FR: Updated empires and descendant records

stepps00 opened the pull request:
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/1027. 

- Updated the [empire of France](https://whosonfirst.mapzen.com/spelunker/id/136253037) to include the geometry of [Wallis and Futuna](https://whosonfirst.mapzen.com/spelunker/id/85632585).
- Updated the geometry of [`Uvea](https://whosonfirst.mapzen.com/spelunker/id/85680901) to include the more precise geometry in [Wallis and Futuna](https://whosonfirst.mapzen.com/spelunker/id/85632585).
- Updated the geometry of [Matâ' Utu](https://whosonfirst.mapzen.com/spelunker/id/890452045) to be on the island of Wallis and Futuna.
- Updated the [empire of Australia](https://whosonfirst.mapzen.com/spelunker/id/136253039/) to include the geometries of: [1](https://whosonfirst.mapzen.com/spelunker/id/404550157), [2](https://whosonfirst.mapzen.com/spelunker/id/404550149), [3](https://whosonfirst.mapzen.com/spelunker/id/404550151), [4](https://whosonfirst.mapzen.com/spelunker/id/874411115)
- Updated the country of [Indian Ocean Territories](https://whosonfirst.mapzen.com/spelunker/id/85632495) to include the geometries of: [1](https://whosonfirst.mapzen.com/spelunker/id/404550157), [2](https://whosonfirst.mapzen.com/spelunker/id/404550149), [3](https://whosonfirst.mapzen.com/spelunker/id/404550151), [4](https://whosonfirst.mapzen.com/spelunker/id/874411115)
- Updated the region of [Cocos (Keeling) Islands](https://whosonfirst.mapzen.com/spelunker/id/85672275) to include the geometries of  [1](https://whosonfirst.mapzen.com/spelunker/id/404550157), [2](https://whosonfirst.mapzen.com/spelunker/id/404550149), [3](https://whosonfirst.mapzen.com/spelunker/id/404550151)

nvkelso commented:
Please make https://whosonfirst.mapzen.com/spelunker/id/85632495 a dependency of Australia versus a country of Australia empire (so the country of Australia is the sold descendent country of the empire). 

See also: 

- https://whosonfirst.mapzen.com/spelunker/id/136253039/descendants/?exclude=nullisland&placetype=dependency

nvkelso commented on data/890/417/321/890417321.geojson:
```diff
@@ -14,11 +15,12 @@
     "qs:photos_9r":"0",
     "qs:photos_sr":"0",
     "qs:pop_sr":"1",
+    "src:geom":"unknown",
```
Seems like `qs` originally...

stepps00 replied:
Yeah, looking at the blame, this was created as part of a QS import.
I'll update this to be `"src:geom":"quattroshapes"`.

nvkelso commented on data/890/452/045/890452045-alt-unknown.geojson:
```diff
@@ -0,0 +1,16 @@
+  "id": 890452045,
+  "type": "Feature",
+  "properties": {
+    "src:geom":"unknown",
```
Is this `qs_pg`?

nvkelso commented on data/874/411/115/874411115.geojson:
```diff
@@ -18,7 +18,7 @@
     "mz:hierarchy_label":1,
-    "mz:is_current":-1,
+    "mz:is_current":1,
     "mz:max_zoom":18.0,
```
This should be `mz:min_zoom` = 6 and mz:max_zoom of 13.

The parent dependency should be `mz:min_zoom` = 4.5  and mz:max_zoom of 7.

nvkelso commented:
Also, implied **Australian** Indian Ocean Territory name should be made explicate, with existing implicate moved to name variant – since these are labeled right next to the British feature that has explicate name.

nvkelso commented:
> Updated the empire of France to include the geometry of Wallis and Futuna.
> Updated the geometry of `Uvea to include the more precise geometry in Wallis and Futuna.
> Updated the geometry of Matâ' Utu to be on the island of Wallis and Futuna.

I don't see updates for these.

stepps00 commented:
Re https://github.com/whosonfirst-data/whosonfirst-data/pull/1028#issuecomment-342985441: 

Both of the following should be in this PR already:
Updated the geometry of `Uvea to include the more precise geometry in Wallis and Futuna.
Updated the geometry of Matâ' Utu to be on the island of Wallis and Futuna.

This was taken care of in commit a021d16832c924636563c8c8fce00e57f3776dd2:
Updated the empire of France to include the geometry of Wallis and Futuna.

nvkelso commented:
Per https://github.com/whosonfirst-data/whosonfirst-data/pull/1028#issuecomment-342984877, can we make use of `label` versus `name` here?

nvkelso commented:
BRB: To action Thanksgiving week.

stepps00 commented:
Comments should be addressed, merge conflicts have been taken care of.

Re https://github.com/whosonfirst-data/whosonfirst-data/pull/1028#issuecomment-343311927, the `wof:name` is "Australian Indian Ocean Territories", with "Indian Ocean Territories" now being a variant name. 

@nvkelso - would you also expect to see  `label:eng_x_preferred` and `label:eng_x_variant` properties, too?

nvkelso commented:
Yes.

stepps00 commented:
@nvkelso - `label` properties added in c68369c; all requested changes should now be taken care of.

nvkelso commented:
Over to @thisisaaronland for PIP work.

stepps00 commented:
@nvkelso - most recent commit fixes conflicts and updates descendant records. Will merge once it gets a second look.

nvkelso commented on data/101/938/925/101938925.geojson:
```diff
@@ -50,15 +52,15 @@
         {
-            "continent_id":-1,
+            "continent_id":102191569,
             "country_id":-1,
```
The "country" (dependency?) here should be Indian Ocean Territories, and empire of Australia? File followup PR to sort that out.

nvkelso replied:
On second thought this should be resolved in this issue as the original issue was related to fixing this type of thing. Just set the right things in the hierarchy and set to controlled property?

nvkelso commented on data/101/938/927/101938927.geojson:
```diff
@@ -83,25 +84,26 @@
         {
             "continent_id":102191569,
-            "country_id":102191569,
+            "country_id":-1,
```
Same deal as `101938925` in https://github.com/whosonfirst-data/whosonfirst-data/pull/1028/commits/b60a235fef5a679afff5057f017a824025e159f4#r196524424.

nvkelso commented on data/136/253/037/136253037.geojson:
```diff
@@ -91,26 +86,14 @@
-        },
-        {
-            "continent_id":102191575,
+            "continent_id":-1,
```
Should the `wof:hierarchy` been set to controlled and retained?

stepps00 replied:
In this case, the `"continent_id":-1,` is correct, but I updated the "wof:parent_id" to be -2 and controlled the hierarchy and parent_id props.

nvkelso commented on data/136/253/037/136253037.geojson:
```diff
@@ -91,26 +86,14 @@
+    "wof:lastmodified":1529359443,
     "wof:name":"France",
-    "wof:parent_id":-2,
+    "wof:parent_id":-1,
```
This should probably still be `-2` per earlier comment about retaining multiple hierarchy.

nvkelso commented on data/404/550/149/404550149.geojson:
```diff
@@ -27,23 +27,26 @@
         {
-            "continent_id":-1,
+            "continent_id":102191569,
             "country_id":-1,
```
Another example of Australia needing better revgeo polygon, see initial comments.

nvkelso commented on data/890/455/929/890455929.geojson:
```diff
@@ -68,16 +67,14 @@
         {
             "continent_id":102191583,
             "country_id":-1,
-            "dependency_id":85632585,
```
We lost the dependency and empire.

stepps00 replied:
added back in via 6727ebc790802

stepps00 commented:
#839 was just merged which included edits to some of these records. 

To do:
- [ ] Reset edits to recently updated features
- [ ] Merge recent commits into this branch
- [ ] PIP all records

stepps00 commented:
https://github.com/whosonfirst-data/whosonfirst-data/pull/1028#issuecomment-342982458 taken care of in https://github.com/whosonfirst-data/whosonfirst-data/pull/1028/commits/fa40d293cd607ad9db1e53dcc9867bf3f898be9f.

stepps00 commented:
Comments addressed in recent commits. Back to @nvkelso for review.
